### ISSUE TITLE
chrome options

### ISSUE THREAD

**Issue by jorisdebock** [jsreport/jsreport#885]
configuring the chrome options, always result in the default values

I have tried on root level, inside extensions as 'chrome' or 'chrome-pdf'

```
{
  "chrome": {
    "strategy": "dedicated-process",
    "numberOfWorkers": 2
  }
  }
```

```
debug: Chrome strategy is chrome-pool, numberOfWorkers: 1
```

**Comment by jorisdebock** [comment]
ah it does work inside 'chrome-pdf' which was a config fault at my side. 

should the config at root level also work? https://jsreport.net/learn/chrome-pdf#chrome-process-allocation

**Comment by pofider** [comment]
Thank you. Yes, it should be possible to configure also from the root chrome.
We will fix that.

**Comment by pofider** [comment]
Fixed in jsreport 3.5.0
